packages/docs/docs/deployment/versioned-builds.md

```diff
@@ -0,0 +1,78 @@
+# Versioned Builds for Source Map Mapping
```

Comment:
litt overkill å ha denne docen? 🤔 Er jo litt kjekk for history og en forklaring på hvorfor vi vil bygge med versjon, men kunne kanskje bare hatt alt dette i PRen istedenfor.. Thoughts?